load_testing/lib/flow_sp_ial2_sign_up.py

```diff
@@ -112,8 +113,23 @@ def ial2_sign_up(context):
         },
     )
 
+    auth_token = authenticity_token(resp)
+
+    resp = do_request(
+        context,
+        "post",
+        "/authentication_methods_setup",
+        "/phone_setup",
+        "",
+        {
+            "_method": "patch",
+            "two_factor_options_form[selection][]": "",
+            "two_factor_options_form[selection][]": "phone",
```

Comment:
ditto, duplicate key?
```suggestion
            "two_factor_options_form[selection][]": "phone",
```